plugin/pkg/scheduler/factory/factory.go

```diff
@@ -365,6 +380,12 @@ func (c *configFactory) invalidatePredicatesForPv(pv *v1.PersistentVolume) {
 	if pv.Spec.AzureDisk != nil {
 		invalidPredicates.Insert("MaxAzureDiskVolumeCount")
 	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeTopologyBinding) {
+		// Add/delete impacts the available PVs to choose from
+		invalidPredicates.Insert("NoVolumeBindConflict")
+	}
+
 	c.equivalencePodCache.InvalidateCachedPredicateItemOfAllNodes(invalidPredicates)
```

Comment:
Question for @kubernetes/sig-scheduling-pr-reviews.  Is the predicate cache per pod per node?  Or just per node?  I was trying to follow this invalidation method and could not find pod references.

Reply:
EquivalenceCache is a 3 level map structure. The first level key is a node name. The second level key is the predicate key, and the last level key is equivalence hash which is extracted from a pod. So, you may not see a reference to a pod directly. You may only see an equivalence hash.